packages/flutter/test/widgets/editable_text_shortcuts_test.dart

```diff
@@ -2201,4 +2208,106 @@ void main() {
     }, variant: appleOnly);
 
   }, skip: !kIsWeb);// [intended] specific tests target web.
+
+  group('Web does accept', () {
+    group('desktop platforms', () {
```

Comment:
Does this nesting make sense @mdebbar?
```
-Web does accept
-- desktop platforms
--- select up
--- select down

... etc
```

Reply:
Is there a need for having the 2nd group (`desktop platforms`)?

Why not:
```
- Web does accept
-- select up
-- select down
... etc
```

Reply:
Yeah probably not - I initially had the groups split up by MacOS and non-macOS because that's how the existing "Web does not accept" tests were setup.  But given that it greatly simplifies the code to just use the ternary operators you suggested, I can just remove the need for that group entirely